### DISCUSSION TITLE
limitations

### DISCUSSION THREAD

**Discussion by AVor0n** [wladradchenko/wunjo.wladradchenko.ru#3] [Q&A] [answered]
Are there watermarks or other restrictions when using?

**Comment by wladradchenko** [comment, marked as answer]
> Are there watermarks or other restrictions when using?

Not. You can use this app without watermarks and limits. Processing time depends on the power of your computer

**Comment by AVor0n** [comment]
Is it possible to use CPU?

**Reply by wladradchenko** [reply]
The default is CPU, but if you install the extension https://github.com/wladradchenko/advanced.wunjo.wladradchenko.ru then you can switch between CPU and GPU.
